CONTRIBUTING.md

```diff
@@ -1,227 +1,150 @@
-```bash
-cd /repository/strapi/packages/strapi-helper-plugin
-npm link
-```
-
-Link the `strapi-helper-plugin` node_modules in the plugin folder:
-```bash
-cd ./my-project/plugins/my-plugin
-npm link strapi-helper-plugin
-```
-
-#### 3. 🚀 Start the project
-
-```bash
-cd ./my-project/admin
-npm run start
-```
-
-The administration panel is available at http://localhost:4000/admin
+#### 5. Available commands
+
+- `yarn watch` starts yarn watch in all packages.
+- `yarn build` builds the `strapi-helper-plugin` (use this command when you develop in the administration panel).
+- `yarn setup` installs the dependencies.
+- `yarn lint` lints the codebase.
+- `yarn test:clean` removes the coverage.
+- `yarn test:front` runs the front-end related tests.
+- `yarn test:front:watch` runs an interactive test watcher for the front-end.
+- `yarn test:snyk` checks the dependencies vulnerabilities.
+- `yarn test:unit` runs the back-end unit tests.
+- `yarn test:e2e` runs an end-to-end test suite.
+- `yarn test:generate-app` generates a test application.
+- `yarn test:start-app` starts the test application.
 
 ---
 
 ## Running the tests
 
 You can run three different kind of tests:
 
-- Unit tests
-- End to end tests
-- Cypress (integration) tests
-
-### Unit tests
-
-This will run the npm `test` script in every packages
-
-```bash
-$ npm run test:unit
-```
-
-### End to end
-
-This will create a test app, start it and run the end to end tests against it.
-
-> ️️️️️️️You can change `test/e2e.js` to configure the database you want to use:
-
-```js
-// test/e2e.js
-//...
-const main = async () => {
-  // you can switch databases.mongo for databases.mysql or any other
-  const database = process.argv.length > 2 ? process.argv.slice(2).join(' ') : databases.mongo;
-  //...
-}
-```
-
-Then run
+**Changing the database:**
 
 ```bash
-$ node test/e2e.js
+$ node test/e2e.js sqlite
+$ node test/e2e.js mongo
+$ node test/e2e.js postgres
+$ node test/e2e.js mysql
 ```
 
-### Cypress
+---
 
-This will create a test app, start it and finally run the cypress tests against it.
+## Additional informations
 
-> You can change `test/createTestApp.js` to configure the database you want to use.
+### Repository Organization
 
-```js
-// test/createTestApp.js
-//...
-const main = async () => {
-  // you can switch databases.mongo for databases.mysql or any other
-  const database = process.argv.length > 2 ? process.argv.slice(2).join(' ') : databases.postgres;
-  //...
-}
-```
-
-> ⚠️ To run the cypress test you must have run `npm run setup:build` to setup the repository
+We made the choice to use a monorepo design such as [React](https://github.com/facebook/react/tree/master/packages), [Babel](https://github.com/babel/babel/tree/master/packages), [Meteor](https://github.com/meteor/meteor/tree/devel/packages) or [Ember](https://github.com/emberjs/ember.js/tree/master/packages) do. It allows the community to easily maintain the whole ecosystem up-to-date and consistent.
```

Comment:
**We chose to** use a monorepo design such as [React](https://github.com/facebook/react/tree/master/packages), [Babel](https://github.com/babel/babel/tree/master/packages), [Meteor](https://github.com/meteor/meteor/tree/devel/packages) or [Ember](https://github.com/emberjs/ember.js/tree/master/packages) do. **This** allows the community to easily maintain the whole ecosystem, **keep it** up-to-date and consistent.